src/modules/launcher/Plugins/Microsoft.Plugin.Uri.UnitTests/UriHelper/ExtendedUriParserTests.cs

```diff
@@ -0,0 +1,50 @@
+using Microsoft.Plugin.Uri.UriHelper;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using NUnit.Framework;
+
+namespace Microsoft.Plugin.Uri.UnitTests.UriHelper
+{
+    [TestFixture]
+    public class ExtendedUriParserTests
+    {
+        [TestCase("google.com", true, "http://google.com/")]
+        [TestCase("localhost", true, "http://localhost/")]
+        [TestCase("127.0.0.1", true, "http://127.0.0.1/")]
```

Comment:
Do we need also the following tests?
- 127.0.0.1:80
- [::]
- [::]:80
- [2001:0DB8::1]
- [2001:0DB8::1]:80